src/software/pipeline/main_incrementalSfM.cpp

```diff
@@ -282,25 +282,32 @@ int main(int argc, char **argv)
 
   if(minNbObservationsForTriangulation < 2)
   {
-      // allows to use to the old triangulatation algorithm (using 2 views only) during resection.
-      minNbObservationsForTriangulation = 0;
-      // ALICEVISION_LOG_ERROR("The value associated to the argument '--minNbObservationsForTriangulation' must be >= 2 ");
-      // return EXIT_FAILURE;
+    // allows to use to the old triangulatation algorithm (using 2 views only) during resection.
+    minNbObservationsForTriangulation = 0;
+    // ALICEVISION_LOG_ERROR("The value associated to the argument '--minNbObservationsForTriangulation' must be >= 2 ");
+    // return EXIT_FAILURE;
   }
   sfmEngine.setNbOfObservationsForTriangulation(minNbObservationsForTriangulation);
 
-  // handle Initial pair parameter
-  if(!initialPairString.first.empty() && !initialPairString.second.empty())
+  // handle initial pair parameter
+  if(!initialPairString.first.empty() || !initialPairString.second.empty())
   {
+    bool isInitialPairCorrect = true;
+    Pair initialPairIndex;
```

Comment:
Need to initialize:
`Pair initialPairIndex = {UndefinedIndexT, UndefinedIndexT};`